pkg/cmd/pulumi/destroy.go

```diff
@@ -159,6 +158,11 @@ func newDestroyCmd() *cobra.Command {
 				return nil
 			}
 
+			cfg, err := getStackConfiguration(s, snap.SecretsManager)
+			if err != nil {
+				return result.FromError(fmt.Errorf("getting stack configuration: %w", err))
+			}
+
```

Comment:
We now get the secretsManager directly from the snapshot that the backend recorded rather than the current project.

unsure if there are edge cases where we want users to be able to have the local directory dictate destroy behavior rather than using the backend's state to unwind.

Reply:
I think we could maybe not load config here for destroy and just default to an empty config map?